episode-6.msrt

```diff
@@ -387,7 +391,8 @@
 78
 00:05:06,020 --> 00:05:10,110
 [ru] [саркастически] Эх, радуйся, Кэррот. Похоже, никто не увидит твое чУдное зелье..
-[en] [sarcastically] You're lucky, Carrot; It seems nobody will see your wondrous potion...
+[en] [Version 1:] [sarcastically] Ah, be glad Carrot. It seems nobody will see your wondrous potion...
```

Comment:
To be a bit more informal, you could say something like "It's your lucky day, Carrot". I agree that the whole thing doesn't flow very well right now. We could change the second part to be something like "We don't have to show your wondrous potion". Just coming up with some ideas.

Maybe we shouldn't focus on Carrot here, after all Pepper's pretty lucky too that she doesn't have to present a bottle of cat piss as her potion. So maybe something like "How fortunate/lucky, it seems..."